### ISSUE TITLE
[1.10.2] Latest Advanced Rocketry doesn't work with latest LibVulpes version

### ISSUE THREAD

**Issue by xXJamieXx** [zmaster587/AdvancedRocketry#582]
The latest Advanced Rocketry version (1.1.7) does not work with the latest LibVulpes version (0.1.18) because it says it needs a version of LibVulpes 0.2.2 or higher. 0.2.2 was an Alpha version and this new latest version is a Release version.

**Comment by zmaster587** [comment]
Advanced Rocketry 1.1.7 (and the whole 1.1.x chain so far) are alpha builds
and will not work with any version of LibVulpes under 0.2.0.  The version
in is correct and enforcement was only recently added due to it being a
point of confusion.

The 1.0.x chain are the stable release builds and will only work with
libvulpes 0.1.x.

I did verify that 1.1.7 is released as alpha on curse's website so I'm
actually kinda curious what site/service is marking it as release.

**Comment by xXJamieXx** [comment]
Yeah I understand that but I'm currently testing Advanced Rocketry 1.1.7 which you are correct is in Alpha.
I'm also using 0.1.18 which is marked as Release and is the latest version. I just wondered why the latest version had a different numbering system to the Alpha builds.

**Comment by zmaster587** [comment]
Libvulpea 0.2.2 was released at the same time as 0.1.18  Though 0.2.3 for
MC 1.7.10 is a version in error and should be also 0.2.2.

The reason the numbees don't march is because LibVulpes and Adanced
Rocketry are 2 separate mods and develop at independent rates.  Libvulpes
just had one more semi large change early on than advanced rocketry which
is why the 2nd number is one larger than AR.

When the alpha build of AR first released libvulpes was already at 0.1.x
but libvulpes also needed some large changes making it incompatible with
earlier builds of AR (adding additional motors, coils, revamp of the mulib
lock upgrade system).  So to indicate a semi major change the minor build
number was increased.

TL;DR. Libvulpes was already on version 0.1.x when AR was on 1.0.x, big
changes to both at the same time required AR go from 0 => 1 and libvulpes 1
=> 2.

**Comment by xXJamieXx** [comment]
But the latest Alpha version of LipVulpes was 0.2.2 and the latest Release version was 0.1.18? 0.1.18 is less than 0.2.2...?

**Comment by xXJamieXx** [comment]
Ok thanks for clearing this up :)

**Comment by zmaster587** [comment]
No problem, and just a point of clarification: 0.1.18 and 0.2.2 were
released at the same time, well , a few minutes apart but essentially the
same time.
